packages/instrumentation-aws-sdk/src/utils.ts

```diff
@@ -14,9 +14,10 @@ export const removeSuffixFromStringIfExists = (str: string, suffixToRemove: stri
 
 export const normalizeV2Request = (awsV2Request: Request<any, any>): NormalizedRequest => {
     const service = (awsV2Request as any)?.service;
+    const operation = (awsV2Request as any).operation;
     return {
-        serviceName: service?.serviceIdentifier?.toLowerCase(),
-        commandName: toCamelCase((awsV2Request as any).operation),
+        serviceName: service?.api?.serviceId,
```

Comment:
Either this or `service?.api?.className` produces the results that we expect, however `className` is a getter and this is already a string so I think this one is better.

Confirmed that it is either `S3` or `DynamoDB` with the right casing.

Reply:
great, I too think this is the property we need